version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     # Use "greater than or equal to" range in modules
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = ">= 1.60.0, < 2.0.0"
+      version = ">= 1.62.0, < 2.0.0"
```

Comment:
same change needed in `modules/fscloud/version.tf`

Reply:
fixed modules/fscloud/version.tf as well - it was missing < 2.0.0